### PR TITLE
348 add unit testing to mentee form

### LINKED ISSUE #348: Add unit testing for MenteeForm.js

### PR DESCRIPTION
**Pull request by katmartel**
I created the tests for the Mentee form without referencing the Mentor form tests. After comparing the two, both have similar tests, just slightly different implementation methods. Upon comparison, I noticed that I hadn't tested for the Time-Zone dropdown, so I used the same code that was used in that test. I was going to make a comment to credit Eric/JP, but I didn't want to add a line of code that might need to be removed before merging.

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]
Nice work

File: src/front-end/src/components/MenteeForm/MenteeForm.test.js
Comment (jpabadir) [inline src/front-end/src/components/MenteeForm/MenteeForm.test.js]: Generally speaking, I wouldn't include a verb in a variable name. This should be `labels` instead of `getLabels`, IMO. 

Edit: I get why you put getLabels. I think you're trying to express in the var name that those are the labels that got rendered rather than the ones we expect, right? Im that case, I'd use an adjective or a past-tense-verb, like maybe `receivedLabels`, `obtainedLabels`, or `renderedLabels`.

File: src/front-end/src/components/MenteeForm/MenteeForm.test.js
Comment (jpabadir) [inline src/front-end/src/components/MenteeForm/MenteeForm.test.js]: Why not a for loop? I'd change this to 
```suggestion
  for (let i = 0; i < getLabels.length; i += 1) {
    expect(getLabels[i].textContent).toBe(correctLabels[i]);
  }
```

**Review by jpabadir — APPROVED** [review]
Good PR! I put a couple of suggestions you may or may not want to include before merging. Your call. 

Btw, I appreciate your intention to credit us for the timezone testing code, but don't worry about it. I think I remember we signed away any rights to this code in the IP agreement ;).

**Comment by katmartel** [comment]
> Good PR! I put a couple of suggestions you may or may not want to include before merging. Your call.
> 
> Btw, I appreciate your intention to credit us for the timezone testing code, but don't worry about it. I think I remember we signed away any rights to this code in the IP agreement ;).

Very true about the waived rights from the IP agreement! But, it felt very wrong to copy and paste code directly, so I wanted to do my due diligence in a comment to recognize your guys' work!!

File: src/front-end/src/components/MenteeForm/MenteeForm.test.js
Comment (katmartel) [reply src/front-end/src/components/MenteeForm/MenteeForm.test.js]: Oh man. It *somehow* completely slipped my mind that in a for loop you can write 'i += 1'. I'm so used to ++ notation, that when I received the prettier warning that ++ notation isn't accepted, my brain decided that the while loop was the way to go. Whoops.

**Pull request merged**
